pkg/prometheus/resource_selector.go

```diff
@@ -1227,6 +1227,9 @@ func (rs *ResourceSelector) validateAzureSDConfigs(ctx context.Context, sc *moni
 
 func (rs *ResourceSelector) validateOpenStackSDConfigs(ctx context.Context, sc *monitoringv1alpha1.ScrapeConfig) error {
 	for i, config := range sc.Spec.OpenStackSDConfigs {
+		if strings.ToLower(config.Role) == "loadbalancer" && rs.version.LT(semver.MustParse("3.2.0")) {
```

Comment:
Hi @dongjiang1989 since the LB is available from the [3.2.0-rc.1](https://github.com/prometheus/prometheus/releases/tag/v3.2.0-rc.1), should we consider that in the must parse?

Reply:
I think its fine to not take release candidates into account TBH.

Reply:
3.2.0 is released and added already in operator 🙂

Reply:
> I think its fine to not take release candidates into account TBH.

+1